infra/base-images/base-builder/precompile_centipede

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Precompiling centipede"
+
+# Add C++15.
```

Comment:
Do you mean Clang 15? 

Also remind me: is Clang 15 strictly necessary here?

Reply:
Yes, I meant Clang 15, sorry about the confusion. Fixed it in the new commit.

`Clang-15` is only needed for dataflow signal tracing. If we don't add `-fsanitize-coverage=trace-loads`, `Centipede` should work just fine (let me double-check this if we want to get rid of `clang-15`).

Reply:
Weird, the documentation for Clang 14 also includes `trace-loads`:

https://releases.llvm.org/14.0.0/tools/clang/docs/SanitizerCoverage.html#tracing-data-flow

And we should be on clang 14?